demo/c/koala_demo_mic.c

```diff
@@ -92,18 +92,24 @@ static void print_dl_error(const char *message) {
 static volatile bool is_interrupted = false;
 
 static struct option long_options[] = {
-        {"access_key",           required_argument, NULL, 'a'},
-        {"audio_device_index",   required_argument, NULL, 'd'},
-        {"library_path",         required_argument, NULL, 'l'},
-        {"model_path",           required_argument, NULL, 'm'},
-        {"output_audio_path",    required_argument, NULL, 'o'},
-        {"reference_audio_path", no_argument,       NULL, 'r'},
-        {"show_audio_devices",   no_argument,       NULL, 's'},
+        {"access_key",              required_argument, NULL, 'a'},
+        {"audio_device_index",      required_argument, NULL, 'd'},
+        {"library_path",            required_argument, NULL, 'l'},
+        {"model_path",              required_argument, NULL, 'm'},
+        {"device",                  required_argument, NULL, 'y'},
+        {"output_audio_path",       required_argument, NULL, 'o'},
+        {"reference_audio_path",    no_argument,       NULL, 'r'},
+        {"show_audio_devices",      no_argument,       NULL, 's'},
+        {"show_inference_devices",  no_argument,       NULL, 'z'},
 };
 
 static void print_usage(const char *program_name) {
     fprintf(stdout,
-            "Usage: %s [-s] [-l LIBRARY_PATH -m MODEL_PATH -a ACCESS_KEY -d AUDIO_DEVICE_INDEX -o WAV_OUTPUT_PATH -r WAV_REFERENCE_PATH]\n",
+            "Usage: %s -l LIBRARY_PATH [-m MODEL_PATH -a ACCESS_KEY -y DEVICE -d AUDIO_DEVICE_INDEX -o WAV_OUTPUT_PATH -r WAV_REFERENCE_PATH]\n"
+            "        %s [-s, --show_audio_devices]\n"
+            "        %s [-z, --show_inference_devices]\n",
```

Comment:
since the library is required for show inference devices
```suggestion
            "        %s [-z, --show_inference_devices] -l LIBRARY_PATH\n",
```